vyaguta-okr-core/src/main/java/com/lftechnology/vyaguta/okr/util/LocalDateTimeDeserializer.java

```
@@ -0,0 +1,29 @@
+package com.lftechnology.vyaguta.okr.util;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+public class LocalDateTimeDeserializer extends JsonDeserializer<LocalDateTime> {
+    private final static DateTimeFormatter DATETIME_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+    @Override
+    public LocalDateTime deserialize(JsonParser jp, DeserializationContext ctxt) throws IOException, JsonProcessingException {
+        if (jp.hasCurrentToken()) {
+            try {
+                LocalDateTime dateTime = LocalDateTime.parse(jp.getValueAsString(), DATETIME_FORMAT);
+                return dateTime;
+            } catch (DateTimeParseException e) {
```

dont swallow exception. log it